docs/pages/about.tsx

```diff
@@ -848,17 +849,19 @@ function AboutContent() {
 
 export default function About() {
   return (
-    <BrandingCssVarsProvider>
-      <Head
-        title="About us - MUI"
-        description="Our mission is to empower anyone to build UIs, faster. We're reducing the entry barrier, making design skills accessible."
-      />
-      <AppHeaderBanner />
-      <AppHeader />
-      <main id="main-content">
-        <AboutContent />
-      </main>
-      <AppFooter />
-    </BrandingCssVarsProvider>
+    <ThemeProvider>
+      <BrandingCssVarsProvider>
```

Comment:
Seems like we can now merge these two components into one, to ease up the implementation. They are used together everywhere.